interface/themes/tabs_style_full.css

```diff
@@ -157,6 +164,15 @@ body
 {
     background-color: #f4f4f4;
 }
+
+#attendantData .patientDataColumn:nth-child(1) span{
+    float:left;
+}
+
+#attendantData .patientDataColumn:nth-child(1) .patientInfo:nth-child(2) {
+    float: left;
+}
+
 .patientInfo a {
```

Comment:
Also need to add these changes to interface/themes/tabs_style_compact.css

Reply:
Done.